Lib/fontbakery/profiles/gdef.py

```diff
@@ -8,13 +8,17 @@
 
 from fontTools import unicodedata
 
+
+GDEF_MARK_GLYPH_CLASS = 3
```

Comment:
Thanks! But this should ideally be placed at `Lib/fontbakery/constants.py` instead.

Reply:
and then imported to be used in the checks of this file